pkg/builder/builder.go

```diff
@@ -695,17 +728,19 @@ func buildPodSpec(vdb *vapi.VerticaDB, sc *vapi.Subcluster) corev1.PodSpec {
 }
 
 // buildScrutinizePodSpec creates a PodSpec for the scrutinize pod
-func buildScrutinizePodSpec(vscr *v1beta1.VerticaScrutinize) corev1.PodSpec {
+func buildScrutinizePodSpec(vscr *v1beta1.VerticaScrutinize, vdb *vapi.VerticaDB, args []string) corev1.PodSpec {
 	termGracePeriod := int64(scrutinizeTermGracePeriod)
 	return corev1.PodSpec{
 		NodeSelector:                  vscr.Spec.NodeSelector,
 		Affinity:                      GetK8sAffinity(vapi.Affinity(vscr.Spec.Affinity)),
 		Tolerations:                   vscr.Spec.Tolerations,
-		InitContainers:                makeScrutinizeInitContainers(vscr),
+		InitContainers:                makeScrutinizeInitContainers(vscr, vdb, args),
 		Containers:                    []corev1.Container{makeScrutinizeMainContainer(vscr)},
```

Comment:
The main container continues to use busybox. Is that the best container to use? I noticed it doesn't have bash, curl or any package manager. Should we find an image that does have these things? Are there other things it needs? Maybe an annotation to allow the default image to be overridden too.

Reply:
Maybe rockylinux:8?